lib/rubocop/cop/rspec/rails/http_status.rb

```diff
@@ -205,7 +205,7 @@ def number
             end
 
             def normalize_str
-              normalized = node.source.delete('"')
+              normalized = node.source.gsub(/["']/, '')
```

Comment:
One can still get away with ‘%[200]’, heredoc etc? Is there a common approach to get the contents of a string?

Reply:
I meant the ‘value’ method instead of regexps on ‘source’.
Example https://github.com/rubocop/rubocop-rspec/blob/865ea173b61cbb0a902cf478e0b24bca7c5751e9/lib/rubocop/cop/rspec/context_wording.rb#L94

Do you think the same approach will be usable here?

Reply:
Ah, I see. Thank you very much. How about this?